skuba-update/skuba_update/skuba-update.timer

```diff
@@ -3,9 +3,9 @@ Description=Daily update of the system
 After=network-online.target local-fs.target
 
 [Timer]
+OnActiveSec=0
```

Comment:
Yes, please saw comment https://github.com/SUSE/skuba/pull/894#pullrequestreview-331765706.

We want to do

> skuba when finished bootstrapping/joining calls skuba-update by using systemd, so it will annotate the node immediately after. This will also make skuba-update annotate the nodes immediately with no updates, while at the moment they are <none> until skuba-update is run the first time by the timer.